src/main/java/com/minecolonies/coremod/commands/colonycommands/ClaimChunksCommand.java

```diff
@@ -0,0 +1,67 @@
+package com.minecolonies.coremod.commands.colonycommands;
+
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.commands.ActionMenu;
+import com.minecolonies.coremod.commands.IActionCommand;
+import com.minecolonies.coremod.util.TeleportToColony;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.util.TeleportToColony'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)